app/helpers/simulations_helper.rb

```diff
@@ -0,0 +1,2 @@
+module SimulationsHelper
+end
```

Comment:
Don't think you need this if you don't implement anything in the helper that's used in the views.